modules/odc-installing-helm-charts.adoc

```diff
@@ -19,7 +19,16 @@ image::odc_helm_chart_devcatalog.png[]
 +
 . In the *Install Helm Chart* page:
 .. Enter a unique name for the release in the *Release Name* field.
-.. Optionally, in the YAML editor, modify the YAML file.
-. Click *Install* to create a Helm release. You will be redirected to the *Topology* view where the release is displayed. If the Helm chart has release notes, the chart is pre-selected and the right panel displays the release notes for that release.
+.. Select the required chart version from the *Chart Version* drop-down list.
++
+.Select chart version
+image::odc_helm_chart_select_chart_ver.png[]
++
+.. Use the *Form view* or the *YAML view* to configure your Helm chart.
++
+[NOTE]
+Some of the Helm Charts may not have the *Form View* option available. Where available, you can switch between the *YAML view* and *Form view*, the data is persisted between the views.
++  
+.. Click *Install* to create a Helm release. You will be redirected to the *Topology* view where the release is displayed. If the Helm chart has release notes, the chart is pre-selected and the right panel displays the release notes for that release.
 
 If required, you can now use the *Actions* button on the side panel or right-click the Helm release to upgrade, rollback, or uninstall the Helm release.
```

Comment:
You could minimise this sentence. For example:

"You can upgrade, rollback, or uninstall a Helm release by using the **Actions** button on the side panel or by right-clicking a Helm release."